src/main.rs

```diff
@@ -4,6 +4,9 @@
 #![allow(unused_mut)]
 #![allow(non_snake_case)]
 #![allow(redundant_semicolons)]
+//#[global_allocator]
+//static GLOBAL: jemallocator::Jemalloc = jemallocator::Jemalloc;
```

Comment:
Let's remove these two commented-out lines too, currently pinning the allocator is not all that relevant

Reply:
done